zerver/webhooks/github/tests.py

```diff
@@ -143,14 +143,14 @@ def test_issue_msg_with_custom_topic_in_url(self) -> None:
         self.check_webhook("issues", expected_topic, expected_message)
 
     def test_issue_assigned(self) -> None:
-        expected_message = "sbansal1999 assigned [issue #7](https://github.com/sbansal1999/testing-gh/issues/7) (assigned to sbansal1999):\n\n~~~ quote\nIDK how this works, lets see if it can be fixed or not.\n~~~"
+        expected_message = "sbansal1999 assigned [issue #7](https://github.com/sbansal1999/testing-gh/issues/7) (assigned to sbansal1999)."
         expected_topic = "testing-gh / issue #7 Sample Issue testing something"
         self.check_webhook("issues__assigned", expected_topic, expected_message)
 
     def test_issue_assigned_with_custom_topic_in_url(self) -> None:
         self.url = self.build_webhook_url(topic="notifications")
         expected_topic = "notifications"
-        expected_message = "sbansal1999 assigned [issue #7 Sample Issue testing something](https://github.com/sbansal1999/testing-gh/issues/7) (assigned to sbansal1999):\n\n~~~ quote\nIDK how this works, lets see if it can be fixed or not.\n~~~"
+        expected_message = "sbansal1999 assigned [issue #7 Sample Issue testing something](https://github.com/sbansal1999/testing-gh/issues/7) (assigned to sbansal1999)."
```

Comment:
This string reads very oddly; I would expect something that reads like a normal sentence, like "{userA} assigned [issue](link) to {userB}", at least in the case that the issue now has one assignee. And what happens if multiple users are assigned, or a user is unassigned? We should test for that and make sure something reasonable happens.

Reply:
Sounds good!